src/bournemouth/resources.py

```
@@ -35,7 +35,8 @@
     stream_chat_response,
 )
 from .models import Message, MessageRole, UserAccount
-from .openrouter import ChatMessage, Role, StreamChoice, StreamChunk
+from .resource_helpers import get_api_key
```

Pylint (3.3.7)

[convention] Import "from .resource_helpers import get_api_key" should be placed at the top of the module

(C0413)